website/client-old/js/services/analyticsServices.js

```diff
@@ -98,6 +98,38 @@
   }
 
   function _gatherUserStats(user, properties) {
+    // filter out animal ears from owned items
+    function _filterAnimalEars (owned) {
+      var animalEarsRegEx = new RegExp('headAccessory_special_[a-z]+Ears', 'g');
+      return Object.keys(owned).reduce(function(animalEars, item) {
+        if (item.match(animalEarsRegEx)) {
+          animalEars.push(item);
+        }
+        return animalEars;
+      }, []);
+    }
+
+    // filter out gem-purchased backgrounds from backgrounds
+    function _filterBackgrounds (owned) {
+      // current free backgrounds
+      var free = ['blue', 'green', 'purple', 'red', 'yellow'];
```

Comment:
Although since the violet one is given when you join the site, it can't ever be purchased, so it doesn't matter that the code won't prevent it being reported in Amplitude. It also won't be a disaster if other free backgrounds are added in future and we forget to update this code. Changing it to identify free backgrounds automatically might be more trouble than it's worth.